### DISCUSSION TITLE
Scores, accuracy, pp and others are not updating

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#16649] [Q&A]
I've been sing osu lazer and it is very much better than the stable, but after completing beat maps it doesn't update the scores on my profile [ not just scores, nothing on the profile is updated ] any way to fix this ?. Im using lazer on MacBook Pro m1 processor, 8gb ram

**Comment by peppy** [comment]
Profile updates are currently limited to recent scores, playcount, and general overall statistics. The rest will be coming soon.

**Reply by ghost** [reply]
> Profile updates are currently limited to recent scores, playcount, and general overall statistics. The rest will be coming soon.

oh, but even they are not updating
